android-sdk/src/main/java/com/mobilecoin/lib/TxOutStore.java

```diff
@@ -37,7 +37,7 @@
 import java.util.stream.Collectors;
 
 final class TxOutStore implements Serializable {
-    private static final String TAG = TxOutStore.class.getName();
+     static final String TAG = TxOutStore.class.getName();
```

Comment:
* why was the visibility changed?
* lint: extra space before static

Reply:
The visibility change might be a question for Sam. I checked usages and it doesn't seem to be used outside of TxOutStore.java. So maybe he meant to change it back. Setting it as private again doesn't seem to break anything, so if the build passes I'll merge.